README.md

```diff
@@ -223,10 +224,10 @@ see [Using the Greynir Shell](docs/shell.md).
 
 ## Copyright and licensing
 
-Greynir is *copyright (C) 2019 by Miðeind ehf.*
+Greynir is *copyright (C) 2020 [Miðeind ehf.](https://mideind.is)*
 The original author of this software is *Vilhjálmur Þorsteinsson*.
 
-![GPLv3](https://raw.githubusercontent.com/mideind/Greynir/master/static/img/GPLv3.png)
+<img src="static/img/GPLv3.png" align="right" style="margin-left:20px;">
```

Comment:
I'm not sure that this works on PyPi, which also displays the README.md file

Reply:
On PyPi?